backend/DataAccess/Registry/ServiceCollectionExtensions.cs

```diff
@@ -11,7 +11,7 @@ public static class ServiceCollectionExtensions
         public static IServiceCollection AddDataAccess(this IServiceCollection services, IConfiguration configuration)
         {
             return services.AddDbContext<BackendContext>(options =>
-                options.UseSqlServer(configuration.GetConnectionString(nameof(BackendContext))));
+                options.UseSqlServer(configuration.GetConnectionString(nameof(BackendContext)), b => b.MigrationsAssembly("RestAPI")));
```

Comment:
Could you explain this change

Reply:
Yes. This is because we changed backend structure. This line tells which of the projects is the one to contain the migration assembly. It is necessary to make migrations work